testing/testing.go

```diff
@@ -13,6 +14,8 @@ import (
 )
 
 var (
+	networkSelected = flag.Int("network", 777, "-network=NETWORKID to select network used for tests")
```

Comment:
That's what I meant. Make the flag a string and a "3" or a "rinkeby" would both make `GetNetworkID()` return `params.RinkebyNetworkID`.

Reply:
I did this previously but @tiabc said he prefered numbers only.

Reply:
@influx6 you shouldn't change anything just because I'd prefer another way, please, only address arguments. And the arguments was that is'd be consistent to `statusd` cli.
@themue feel free to argue.

Reply:
IC, okay, thanks for info.

Reply:
That's why I meant to accept both, number as well as identifier. Using the number as caller it would be consistent to `statusd`.  And using a string as caller it would be more readable and better to keep in mind when forget the network IDs.

As a follow-up issue we could make the same change to `statusd`. So todays calls wouldn't be incompatible but also here a more speaking call could be used.

Reply:
@tiabc I see, understood. I set the usage of names as id because it's more convenient and even user experience makes more sense, no one naturally remembers the chain network id but all would equally remember the name. Will make updates to this once am done with #420.

Reply:
I don't have a strong opinion here and don't think it's worth discussing currently so my first thought was to use an already existing format.